src/main/java/microservices/userServiceBasic/api/response/UserResponse.java

```diff
@@ -0,0 +1,29 @@
+package microservices.userServiceBasic.api.response;
+
+
+import microservices.userServiceBasic.api.model.UserDetails;
+
+import java.util.UUID;
+
+public class UserResponse extends UserDetails {
```

Comment:
Hmm... these DTOs should be independent from each other. I know the data is similar for now, but the whole point is to decouple the code. So it's better to add duplicated data here and not extend.